terraform/main.tf

```diff
@@ -0,0 +1,139 @@
+terraform {
+  backend "gcs" {
+    bucket = "f3441e415e6e5e7d-bucket-tfstate"
+    prefix = "terraform/state"
+  }
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.51.0"
+    }
+  }
+}
+
+variable "project_id" {
+  type    = string
+  default = "catalyst-cooperative-pudl"
```

Comment:
Is there anything weird about using the PUDL project from a different repo? Should we just create a mozilla project?

Reply:
If we're billing infrastructure costs to Mozilla, we should probably make a separate mozilla project - that will make the bookkeeping much easier.

If not, I think it's fine to use the same GCP project - though if we do that, I'd rather this terraform configuration live in the `pudl` repo. I think one `tfstate` per GCP project is easier to think about.